specs/waku/v2/waku-relay.md

```diff
@@ -22,11 +31,39 @@ authors: Oskar Thorén <oskar@status.im>
 
 **Protocol identifier***: `/vac/waku/relay/2.0.0-beta2`
 
+## Security Requirements
```

Comment:
Should this really be part of the abstract? right now the way the document is set up it is.

Reply:
Thanks for pointing this out! Apparently, everything in the relay specs is under the abstract section, will fix it.